packages/framework/src/Model/Category/Category.php

```diff
@@ -210,11 +214,114 @@ protected function setTranslations(CategoryData $categoryData)
         }
     }
 
+    /**
+     * @param int $domainId
+     * @return string|null
+     */
+    public function getSeoTitle($domainId)
```

Comment:
In previous entities these methods were always type-hinted as `int`.

Reply:
added type hint